src/tokens/Assets.tsx

```diff
@@ -572,6 +587,11 @@ const styles = StyleSheet.create({
     ...typeScale.labelMedium,
     color: Colors.dark,
   },
+  topBarTextItem: {
```

Comment:
Is it intentional that the style for `TextButton` is named `TextItem`? I'm doubtful that the `greenUI` color could be reusable for regular text :sweat_smile:

Reply:
Got distracted by `tabBarItem` as it sounds close to `topBar...`. Fixed.

TopBarTextButton actually uses `colors.greenUI` by default. Removed here due to repeating it.